specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2019-05-05-preview/AlertsManagement.json

```diff
@@ -655,11 +655,7 @@
       "enum": [
         "asc",
         "desc"
-      ],
-      "x-ms-enum": {
-        "name": "sortOrder",
-        "modelAsString": true
-      }
+      ]
```

Comment:
Please add this back, but please make sure the new name of this enum type should have its first letter capitalized. `SortOrder` for instance, or `SortOrderType`
Please feel free to suggest a type name which is using in Java SDK now @weidongxu-microsoft

Reply:
Hi, I changed this to SortOrder and added it to the common swagger since it created duplicate parameter error.
It created some breaking changes, could you please take a look at it?